webapp/channels/src/components/onboarding_tasklist/onboarding_tasklist_popover.tsx

```diff
@@ -49,12 +48,12 @@ const Overlay = styled.div`
 `;
 
 interface TaskListPopoverProps {
-    trigger: RefObject<HTMLButtonElement>;
+    trigger: HTMLButtonElement | null;
```

Comment:
Why not keep the `RefObject` and just call the `current` value whenever needed?

Reply:
See above. We need this to re-render so that the `useFloating` can calculate the position of this component